src/sentry/static/sentry/app/components/activity/item/index.tsx

```diff
@@ -154,6 +154,7 @@ const ActivityFooter = styled(HeaderAndFooter)`
 
 const ActivityBody = styled('div')`
   padding: ${space(2)} ${space(2)};
+  word-break: break-all;
```

Comment:
This can slice words in half in firefox, but other values of word-break have different/worse problems.

Reply:
Yeah, I looked into the other options and they also weren't compatible with more browsers than this option 😬